src/mpl/include/mpl_thread_posix.h

```diff
@@ -14,7 +14,14 @@
 #include <errno.h>
 #include <pthread.h>
 
+#define MPL_POSIX_MUTEX_DEFAULT 0
+#define MPL_POSIX_MUTEX_TICKETLOCK 1
+
+#if MPL_POSIX_MUTEX_NAME == MPL_POSIX_MUTEX_TICKETLOCK
 #include "mpl_posix_mutex_native.h"
+#else
+#include "mpl_posix_mutex_ticketlock.h"
+#endif
```

Comment:
This is incorrect.  You are pointing ticketlock to native.

Reply:
Thanks. I was rushing it :). Now I know it is in the right direction, I'll take my time fixing it.

Reply:
fixed.